### ISSUE TITLE
cxxopts 2.1.2 does not compile in VS 2019 Update 2.

### ISSUE THREAD

**Issue by fschoenm** [jarro2783/cxxopts#193]
After updating to Visual Studio 16.2.0, cxxopts generates the following compile error:

```
7>...\include\cxxopts.hpp(349,39): error C2677:  binary '+': no global operator found which takes type 'const std::string' (or there is no acceptable conversion)
7>...\include\cxxopts.hpp(349,26): error C2512:  'cxxopts::OptionSpecException::OptionSpecException': no appropriate default constructor available
```

**Comment by jarro2783** [comment]
Can you give me some sample code that causes this, and the compile flags that you have used?

**Comment by fschoenm** [comment]
Sorry, this seems to be already fixed in 2.2.0 (with #172). I wasn't aware of the new release.
